subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.h

```diff
@@ -90,6 +90,10 @@ struct lll_adv {
 #if defined(CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL)
 	s8_t tx_pwr_lvl;
 #endif /* CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL */
+
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
```

Comment:
Remove this code block for the reasons state before.

(Side note, the BT Spec needs only one octet for counts)